src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial.mdx

```diff
@@ -0,0 +1,480 @@
+---
+title: "Introductory NRQL tutorial: unlock your data"
+tags:
+  - NRQL tutorial
+  - 'NRQL: New Relic query language'
+  - Get started
+  - Tutorial
+translate:
+  - jp
+  - kr
+metaDescription: "Learn how to use NRQL with this introductory tutorial."
+redirects:
```

Comment:
You can also leave redirects out if there are no redirects

Reply:
Thank you!